src/styles/blocks/compare.scss

```diff
@@ -0,0 +1,47 @@
+.compare {
+  &__content {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: minmax(auto, 392px);
```

Comment:
I generally don't use grid for mobile. 

But this code also works without any flaw.

Reply:
I added this because bicycle pictures were getting so big right before hitting the tablet breakpoint. Then getting smaller again in tablet view.  And when I add max-width pictures weren't responsive anymore. I am open to suggestions if there is a better way to do it.

Reply:
No, I don't think anything wrong with this